moncli/entities/update.py

```diff
@@ -259,6 +259,7 @@ def get_replies(self, *args):
         while record_count >= page_limit:
             updates_data = api.get_updates(
                 'id', 'item_id', 
+                api_key=self.__creds.api_key_v2, 
```

Comment:
This was something from the merge conflict. Please remove this line.

Moving forward, please be sure that all key-word arguments are below any args lists for visual consistency.